libyul/optimiser/DataFlowAnalyzer.cpp

```diff
@@ -84,13 +84,26 @@ void DataFlowAnalyzer::operator()(Switch& _switch)
 
 void DataFlowAnalyzer::operator()(FunctionDefinition& _fun)
 {
+	// Save all information. We might rather reinstantiate this class,
+	// but this could be difficult if it is subclassed.
+	map<string, Expression const*> value;
```

Comment:
Will this code be changed to use YulString?

Reply:
Yep, will have to.